specs/phase1/custody-game.md

```diff
@@ -229,9 +229,11 @@ Given one set of data, return the custody atoms: each atom will be combined with
 
 ```python
 def get_custody_atoms(bytez: bytes) -> Sequence[bytes]:
-    bytez += b'\x00' * (-len(bytez) % BYTES_PER_CUSTODY_ATOM)  # right-padding
-    return [bytez[i:i + BYTES_PER_CUSTODY_ATOM]
-            for i in range(0, len(bytez), BYTES_PER_CUSTODY_ATOM)]
+    bytez += b'\x00' * ((BYTES_PER_CUSTODY_ATOM - len(bytez)) % BYTES_PER_CUSTODY_ATOM)  # right-padding
```

Comment:
it's possible that `len(bytez) > BYTES_PER_CUSTODY_ATOM`, so `(BYTES_PER_CUSTODY_ATOM - len(bytez))` can become negative

Reply:
Ah you're right. Hmm so it can be:
```suggestion
    length_remainder = len(bytez) % BYTES_PER_CUSTODY_ATOM
    bytez += b'\x00' * ((BYTES_PER_CUSTODY_ATOM - length_remainder) % BYTES_PER_CUSTODY_ATOM)  # right-padding
```

Reply:
That looks correct!
I wrote a small test and it works fine.
```python
from eth2spec.phase1.spec import BYTES_PER_CUSTODY_ATOM

for i in range(0, BYTES_PER_CUSTODY_ATOM*2+1):
    bytez = b'\0' * i
    length_remainder = len(bytez) % BYTES_PER_CUSTODY_ATOM
    bytez1 = bytez + (b'\x00' * ((BYTES_PER_CUSTODY_ATOM - length_remainder) % BYTES_PER_CUSTODY_ATOM))
    bytez2 = bytez + (b'\x00' * (-len(bytez) % BYTES_PER_CUSTODY_ATOM))
    assert bytez1 == bytez2
```